### PR TITLE
Remove sync time hardcoding

### PR DESCRIPTION
**Pull request by leizhiyuan**
Remove sync time hardcoding

### CODE REVIEW

File: pilot/cmd/pilot-discovery/main.go [hunk 1]
```diff
@@ -112,8 +112,12 @@ func init() {
 		"Consul Config file for discovery")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Service.Consul.ServerURL, "consulserverURL", "",
 		"URL for the Consul server")
+	discoveryCmd.PersistentFlags().DurationVar(&serverArgs.Service.Consul.Interval, "consulserverInterval", 2*time.Second,
+		"Interval for the Consul server")
```
Comment (nmittler) [inline pilot/cmd/pilot-discovery/main.go]: Can we make the comment a little more clear .... maybe something like "Interval (in seconds) for polling the Consul service registry"?

File: pilot/cmd/pilot-discovery/main.go [hunk 2]
```diff
@@ -112,8 +112,12 @@ func init() {
...
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Service.Eureka.ServerURL, "eurekaserverURL", "",
 		"URL for the Eureka server")
+	discoveryCmd.PersistentFlags().DurationVar(&serverArgs.Service.Eureka.Interval, "eurekaserverInterval", 2*time.Second,
+		"Interval for the Eureka server")
```
Comment (nmittler) [inline pilot/cmd/pilot-discovery/main.go]: Same comment as above.

File: pilot/cmd/pilot-discovery/main.go (hunk 1 above)
Comment (leizhiyuan) [reply pilot/cmd/pilot-discovery/main.go]: @nmittler Thanks for your advice, I have fixed it.

**Comment by leizhiyuan** [comment]
* (auth infra) http-reachability run 0 failed all 90 attempts for Checking log of mixer

**Comment by leizhiyuan** [comment]
oh.. when tryed again,  e2e-pilot has passed.

**Comment by leizhiyuan** [comment]
@andraxylia  what should I do? Sometimes the tests passed. but sometimes don't pass.. I think It was because the tests are not unstable. I can not make them pass once..

**Pull request merged**